packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -183,6 +184,26 @@ export async function $onEmit(context: EmitContext<NetEmitterOptions>) {
   }
 }
 
+/**
+ * Write the code model to the output folder.
+ * @param program - The program
+ * @param codeModel - The code model
+ * @param outputFolder - The output folder
+ * @param fileName - The file name
+ * @beta
+ */
+export function writeCodeModel(
+  program: Program,
+  codeModel: CodeModel,
+  outputFolder: string,
+  fileName: string,
+) {
+  program.host.writeFile(
+    resolvePath(outputFolder, fileName),
+    prettierOutput(JSON.stringify(codeModel, null, 2)),
```

Comment:
is this function correctly implemented? this is not stringify the json with refs.

Reply:
Good catch - this is the config implementation.